res/TensorFlowLiteRecipes/Inf_Pad_000/test.rule

```diff
@@ -0,0 +1,6 @@
+# To check if dynamic dimension properly infered
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "PAD_EXIST"               $(op_count PAD) '=' 1
+RULE    "PAD_SHAPE"               $(tensor_shape ofm) '=' 1,-1,7,2
```

Comment:
The 'shape' expression `1,-1,7,2` is fine now, but it would be better to express it in a more formalized form like `[1, -1, 7, 2]`.
It would be better if it could handle spaces between commas as well.

```suggestion
RULE    "PAD_SHAPE"               $(tensor_shape ofm) '=' [1, -1, 7, 2]
```

Reply:
from #https://github.com/Samsung/ONE/issues/13998#issuecomment-2370601917

> FYI) RULE "PAD_SHAPE" $(tensor_shape ofm) '=' [1, -1, 7, 2] does not work with the current code. Currently, the rule only works if the arguments have exactly 4 elements, so there cannot be any spaces in between, such as [1, -1, 7, 2]. Therefore, at the moment, we are checking shapes in the form of 1,-1,7,2.
> 
> * Current code
>   https://github.com/Samsung/ONE/blob/646c7f16511f9baa8384f719e65cba768d163f7b/compiler/dredd-rule-lib/rule-lib.sh#L85-L92

Reply:
Oh, thanks for the description.
How about using a quoted string, i.e., `"[1, -1, 7, 2]"`?
If it's difficult, the current implementation is not bad either =)